reports/Washington.md

```diff
@@ -165,6 +179,20 @@ id: wa-seattle-10
 * Related to OPD Case 2020OPA-0333 "A flashbang striking a reporter"
 
 
+### Police injure sleeping homeless man with flashbang | June 1st
+
+Police flashbang hits a homeless man sleeping in a park.
+
+tags: flashbang, homeless
+
+id: wa-seattle-54
+
+**Links**
+
+* https://www.facebook.com/kg.farmer/videos/278656296661372
+* Related to OPD Case 2020OPA-0335 "Homeless man hit by an SPD flashbang while sleeping"
```

Comment:
I think all of these should go into description rather than Links because they're not links